client/utils/validation.js

```diff
@@ -66,7 +69,23 @@ const rules = {
   url
 };
 
-forEach(rules, (rule, name) => extend(snakeCase(name), {
-  message: messages[name],
-  ...rule
-}));
+processRules(rules).then(registerRules);
```

Comment:
This will still include everything upon build so I am not sure if it makes sense to do it.
@MiroDojkic @abasic?

Reply:
@underscope agreed. The module is [light](https://bundlephobia.com/result?p=@vee-validate/rules@4.1.20) so I think we should be fine with the static import of all the rules.

Reply:
Since module is light, should we register all of them and keep this comparison, or should we register all of them to keep codebase cleaner, thoughts? 🤔  @MiroDojkic